polaris/ocean/tasks/sphere_transport/__init__.py

```diff
@@ -0,0 +1,250 @@
+from typing import Dict
+
+from polaris import Step, Task
+from polaris.config import PolarisConfigParser
+from polaris.ocean.mesh.spherical import add_spherical_base_mesh_step
+from polaris.ocean.tasks.sphere_transport.analysis import Analysis
+from polaris.ocean.tasks.sphere_transport.filament_analysis import (
+    FilamentAnalysis,
+)
+from polaris.ocean.tasks.sphere_transport.forward import Forward
+from polaris.ocean.tasks.sphere_transport.init import Init
+from polaris.ocean.tasks.sphere_transport.mixing_analysis import MixingAnalysis
+from polaris.ocean.tasks.sphere_transport.viz import Viz, VizMap
+
+
+def add_sphere_transport_tasks(component):
+    """
+    Add tasks that define variants of the cosine bell test case
+
+    component : polaris.ocean.Ocean
+        the ocean component that the tasks will be added to
+    """
+
+    for icosahedral, prefix in [(True, 'icos'), (False, 'qu')]:
+        for case_name in ['rotation_2d', 'nondivergent_2d', 'divergent_2d',
+                          'correlated_tracers_2d']:
+            filepath = f'spherical/{prefix}/{case_name}/{case_name}.cfg'
+            config = PolarisConfigParser(filepath=filepath)
+            config.add_from_package('polaris.ocean.convergence.spherical',
+                                    'spherical.cfg')
```

Comment:
```suggestion
            config.add_from_package('polaris.ocean.convergence',
                                    'convergence.cfg')
            config.add_from_package('polaris.ocean.convergence.spherical',
                                    'spherical.cfg')
```

Reply:
A bunch of stuff related to rebasing on #131 got lost even though this PR was based on those commits before. I need to go through the changes again more carefully.